pkg/api/graphql/cells/cells.go

```diff
@@ -0,0 +1,78 @@
+package cells
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	msg "github.com/aziontech/azion-cli/messages/logs/cells"
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/machinebox/graphql"
+)
+
+type CellsConsoleEvent struct {
+	Ts              time.Time `json:"ts"`
+	SolutionId      string    `json:"solutionId"`
```

Comment:
Does JSON come in this kind of formatting?  CamelCase?

Reply:
```
{
        "ts": "2024-01-30T20:11:00Z",
        "solutionId": "1531930033",
        "configurationId": "1706643363",
        "functionId": "16147",
        "id": "c89728ff-7261-4cb8-bf60-d219d0668975",
        "lineSource": "CONSOLE",
        "level": "LOG",
        "line": "Worker responding with: Hello World"
      }
```